### ISSUE TITLE
0/0 should not pass

### ISSUE THREAD

**Issue by jo** [scottcorgan/tap-spec#38]
I had a misconfiguration which caused that no tests were run. This was not detected by the tests:

https://travis-ci.org/eHealthAfrica/couchdb-bootstrap/jobs/76070539#L206

I think that in general zero tests should not pass.

**Comment by scottcorgan** [comment]
What does your TAP output look like with tap-spec removed?

**Comment by jo** [comment]
I am using tape [here](https://github.com/eHealthAfrica/couchdb-bootstrap):

```sh
[jo@localhost couchdb-bootstrap]$ node_modules/.bin/tape test/*.js 
TAP version 13
# configure
/home/jo/github/couchdb-bootstrap/test/helper.js:46
    async.map(Object.keys(config), function(key, next) {
                     ^
TypeError: Object.keys called on non-object
    at Function.keys (native)
    at /home/jo/github/couchdb-bootstrap/test/helper.js:46:22
    at Request._callback (/home/jo/github/couchdb-bootstrap/node_modules/nano/lib/nano.js:192:16)
    at self.callback (/home/jo/github/couchdb-bootstrap/node_modules/nano/node_modules/request/request.js:354:22)
    at Request.emit (events.js:107:17)
    at Request.onRequestError (/home/jo/github/couchdb-bootstrap/node_modules/nano/node_modules/request/request.js:1011:8)
    at ClientRequest.emit (events.js:107:17)
    at Socket.socketErrorListener (_http_client.js:271:9)
    at Socket.emit (events.js:107:17)
    at net.js:459:14

[jo@localhost couchdb-bootstrap]$ echo $?
1
```

**Comment by scottcorgan** [comment]
This is fixed and published as `4.1.0`

**Comment by jo** [comment]
Awesome, thanks!
